content/docs/overview.md

```diff
@@ -779,6 +779,29 @@ foo :: proc{
 }
 ```
 
+### Passing procedures as parameters
```

Comment:
You have put this before "Basic types", which should be moved further ahead.

Reply:
🤔 Yeah, i wanted it to be a subsection under `Procedures`, but Procedures comes before "Basic types". Do you have a recommendation for where it should go instead?